pages/about/groups/task-forces/adapt.md

```diff
@@ -102,6 +102,8 @@ The WAI-Adapt Task Force develops and maintains the following resources for the
 * [WAI-Adapt: Help and Support Module &mdash; Working Draft](https://www.w3.org/TR/adapt-help/) (and latest [Editor’s Draft](https://w3c.github.io/adapt/help/)) &mdash; examples of the personalized help and support properties
 * [Requirements for WAI-Adapt specification &mdash; Group Draft Note](https://www.w3.org/TR/adapt-requirements/) (and latest [Editor’s Draft](https://w3c.github.io/adapt/requirements/) &mdash; user stories, specific use cases, and requirements for WAI-Adapt
 
-## Contact the chair
+## Contact the task force
 
-If you have a question for the WAI-Adapt Task Force, email the [chair](https://www.w3.org/groups/tf/personalization-tf/participants/#chairs) or the [W3C staff contact](https://www.w3.org/groups/tf/personalization-tf/participants/#staff).
\ No newline at end of file
+If you have a question for the WAI-Adapt Task Force, email the [group](mailto:public-adapt@w3.org) (<strong>Note:</strong> This email will go to a publicly archived mailing list).
```

Comment:
```suggestion
If you have a question for the WAI-Adapt Task Force, [email the task force](mailto:public-adapt@w3.org) (<strong>Note:</strong> This email will go to a publicly archived mailing list).
```

As per above.

Also, a TF isn't really a group (in the W3C sense) so refering to it as a TF is a wee bit more appropriate.